client/fingerprint/env_aws_cpuinfo.go

```diff
@@ -0,0 +1,1620 @@
+// Code generated from hashicorpnomad/ec2info; DO NOT EDIT.
```

Comment:
Seeing that this is basically a shell script, I'd include it in the OSS repo with instructions.  One benefit might be getting contributions when the data is stale.